src/Web/Slack.hs

```diff
@@ -78,6 +79,10 @@ import qualified Web.Slack.Im as Im
 import qualified Web.Slack.Group as Group
 import qualified Web.Slack.User as User
 
+#if !MIN_VERSION_servant(0,13,0)
+let mkClientEnv = ClientEnv
```

Comment:
@emmanueltouzery This fails if using Servant < 0.13 since it's in the middle of imports: 

```
.../slack-web/src/Web/Slack.hs:87:1: error:
    parse error (possibly incorrect indentation or mismatched brackets)
   |
87 | import Data.Text (Text)
   | ^
```